### DISCUSSION TITLE
筆電散熱膏更換分享

### DISCUSSION THREAD

**Discussion by yucheng1028** [1101-GS4719-C/Python-Community-GS4719-C#533] [閒聊八卦]
### 筆電更換散熱膏分享

這是一台 2019年 ROG 西風之神 GM501GM 筆電
處理器 i7-8750h 配上 GTX - 1060 顯卡

最近發現溫度表現在接上銀幕後實在不優
甚至會過熱降頻或藍屏

所以買了一條 暴力熊 ( K=12.5 ) 散熱膏嘗試自己更換

換完之後待機溫度由70度降至60度
燒雞溫度待測

如果你有動手能力，筆電又剛好過保的話，可以嘗試看看

_筆電散熱膏推薦_
https://www.mobile01.com/topicdetail.php?f=159&t=5677444

**Comment by Ben6uo** [comment]
70度60度 是閒置情況? 打遊戲70度60度很正常   不過筆電是時候換一換了 直接AMD5950X RTX30走起

**Reply by yucheng1028** [reply]
換之前不知為啥待機CPU都占用20上下，換之前溫度70度，玩遊戲可以到97度....感覺真的要重灌或換電腦了

**Comment by d26488399** [comment]
推薦直接換筆電，簡單又暴力

**Comment by yi-1111** [comment]
打Apex是不是會降頻啊，就不要爬鑽就好了

**Comment by JSHT** [comment]
再不行，上液金 ((沒，一不小心就會整台報廢

**Comment by Sazmafuji** [comment]
我常看到我電腦燒到90度，我都怕電腦燒死了，感謝分享

**Comment by ThomasMeng0515** [comment]
感謝分享，之前也有考慮要用散熱膏，但是不知道效果如何~

**Comment by sean1325418** [comment]
希望不要電腦拆開裝不回去就好

**Comment by bloodtrail0511** [comment]
@d26488399 該換散熱膏了

**Comment by Ku-Hao-Chen** [comment]
謝謝分享，讓我知道筆電散熱膏更換方法。

**Comment by HaoFish123** [comment]
感謝分享，讓我多了些電腦相關知識。

**Comment by leon10456** [comment]
感謝分享 今天也來換桌電的散熱膏

**Comment by jason79461385** [comment]
自己筆電整體溫度也是偏高，近期在高壓的環境下自己筆電也會出現藍屏的情況，感謝分享。

**Comment by alexthebigpp** [comment]
感謝分享，之後有空來換自己比電的散熱糕

**Comment by yenwei12138** [comment]
感謝分享，第一次聽到散熱膏這東西，感覺我家常常高燒不退的筆電也可以試試

**Comment by awkward-willy** [comment]
謝謝分享
不過拆的時候還是要很小心啦
小弟我有一兩個卡點慘死在我粗暴的手法上
